### DISCUSSION TITLE
Why doesn't my sandbox render?

### DISCUSSION THREAD

**Discussion by NullVoxPopuli** [codesandbox/codesandbox-client#4805] [General]
https://codesandbox.io/s/dependency-injection-4-5fjzg?file=/index.html

I'm using the vanilla JS template (which uses parcel)

the preview is just a white/blank page, even though I have tons of HTML in the html file

**Comment by SaraVieira** [comment]
Hey!

You need to install both babel dependencies and it will work: https://codesandbox.io/s/dependency-injection-4-forked-w3e85?file=/package.json:359-370

We don't install dev dependencies in browser sandboxes

**Reply by NullVoxPopuli** [reply]
how did you know to do that?

**Reply by SaraVieira** [reply]
We have had a bunch of issues with babel.in the past

**Reply by NullVoxPopuli** [reply]
is there any logging that I can find or that can be added to help out future folks?

**Reply by NullVoxPopuli** [reply]
unrelated to debugging, it looks like my sandbox still isn't loading somehow?
is there a delete cache or restart button somewhere?

![image](https://user-images.githubusercontent.com/199018/92181523-6dae0100-ee17-11ea-801c-25bc56b87dce.png)

**Reply by NullVoxPopuli** [reply]
if I visit the output URL directly, https://5fjzg.codesandbox.io/ it works fine. o.o

**Reply by NullVoxPopuli** [reply]
the sandboxes load in private browsing. hmm

**Reply by NullVoxPopuli** [reply]
but this one https://codesandbox.io/s/dependency-injection-1-19yqj loads in my normal window, non-private browsing. 

the only difference I can think of is that it doesn't use parcel... but why would that matter? hmm

**Reply by NullVoxPopuli** [reply]
These are the 4 sandboxes:
1. https://codesandbox.io/s/dependency-injection-1-19yqj
2. https://codesandbox.io/s/dependency-injection-2-b0qws
3. https://codesandbox.io/s/dependency-injection-3-mum0p
4. https://codesandbox.io/s/dependency-injection-4-5fjzg


They all work in private browsing mode, but only #1 works in non-private-browsing.

**Reply by SaraVieira** [reply]
That is interesting, they all work for me

May be some sort of cache :/

**Reply by NullVoxPopuli** [reply]
looks like it's related to Privacy Badger

**Reply by NullVoxPopuli** [reply]
![image](https://user-images.githubusercontent.com/199018/92248259-6417ae00-ee96-11ea-8790-ed72a505293f.png)

**Reply by NullVoxPopuli** [reply]
that red one is probably what needs to not be blocked

**Reply by SaraVieira** [reply]
Yeah, that's the sandbox itself, you need to allow that :)

**Reply by NullVoxPopuli** [reply]
sure, but it's not a me problem. lots of people have privacy badger installed. 

I'm curious why privacy badger is blocking csb and if we can ask them to not block csb

**Reply by SaraVieira** [reply]
I mean we have no idea why, it may be because we have had phising issues in some sandboxes

**Reply by NullVoxPopuli** [reply]
I "submitted an error report" so... we'll see what happens?

**Reply by SaraVieira** [reply]
yeah, hope they take it off
